notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>"...this tutorial uses <code>GBTRegressor</code> since it shows the highest score."</p><p> </p><p>You can say "in testing it was shown that the GBT Regressor provides the best results".</p><p> </p><p>Do you have the code handy for some of the others? You can add a commented cell that has the code for these in case the user is interested in experimenting themselves.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>